portafly/src/i18n/locales/en/login.yml

```
@@ -0,0 +1,14 @@
+---
+page_title: Login | Portafly
+page_title_description: Log in to your account
```

```suggestion
page_title_description: Page title of the Login page
```
The descriptions should be useful for the translators and technical writers, giving context

---

Hmm... I didn't put much of effort since indeed those are gonna change. BUT still, it needs to be reviewed first for others in the team that need some other kind of visual cues

---

I know they are gonna change, but we should get used to add them. Besides is a compromise we made with Docs team